start.py

```diff
@@ -1217,14 +1217,13 @@ def restart_callback():
 def start_metrics(m2ee):
     metrics_interval = os.getenv("METRICS_INTERVAL")
     profile = os.getenv("PROFILE")
-    if metrics_interval and profile != "free":
+    if metrics_interval:
         import metrics
-
         thread = metrics.MetricsEmitterThread(int(metrics_interval), m2ee)
         thread.setDaemon(True)
         thread.start()
     else:
-        logger.info("MENDIX-INTERNAL: Metrics are disabled.")
+        logger.info("MENDIX-INTERVAL not set: Metrics are disabled.")
```

Comment:
I wouldn't change this.

Also, the error message doesn't describe what's happening. `MENDIX-INTERVAL` is not what we are looking for, an d the message doesn't mention that this is an environment variable.

Reply:
the earlier log message also did not make sense. I would rather reword it and clarify that `MENDIX-INTERVAL` is now the environment variable that decides whether metrics get pushed or not.

Reply:
`MENDIX-INTERVAL` is _not_ the environment variable that decides this though. (it's `METRICS_INTERVAL`)

We (try to) prefix all non-debug log messages with `MENDIX-INTERNAL` to indicate that they are internal messages. Please leave this as is.